### PR TITLE
Feature #1676 - Allow duplicate sysnames during network discovery

### PR DESCRIPTION
**Pull request by netniV**
Feature #1676 adds functionality to enable duplicate sysnames to be added to the system on multiple hosts.  An option will be visible in the settings of the Network being discovered to enable this functionality which is disabled by default to maintain the current level of functionality

### CODE REVIEW

File: poller_automation.php [hunk 1]
```diff
@@ -579,27 +579,30 @@ function discoverDevices($network_id, $thread) {
 							if ($snmp_sysName != '') {
 								$isCactiSysName = db_fetch_cell_prepared('SELECT COUNT(*)
 									FROM host
+									AND ip == ?
 									WHERE snmp_sysName = ?',
-									array($snmp_sysName));
+									array($snmp_sysName, $device['ip_address']));
```
Comment (cigamit) [inline poller_automation.php]: This seems backwards.

Comment (netniV) [reply poller_automation.php]: Yes, it's the AND statement that is on the wrong line somehow.  Not sure how that happened.  I've pushed through an update.

**Comment by holbor** [comment]
Issue: The checkbox does not stay checked.  I get the pop-up saying successful but then the checkbox goes back to empty.. Does seem to work though.
Also would be SUPER NICE feature to allow using DNS name as device description.
/thanks

**Comment by netniV** [comment]
That would be because you don';t have the field in your database.  If you are already running 1.2 you have to manually apply the database change.

**Comment by holbor** [comment]
Thanks,
ALTER TABLE `automation_networks` ADD COLUMN `same_sysname` char(2);
fixed.

/rh

**Pull request merged**